airbyte-integrations/connectors/source-stripe/pyproject.toml

```diff
@@ -19,7 +19,7 @@ include = "source_stripe"
 python = "^3.10,<3.12"
 stripe = "==2.56.0"
 pendulum = "==2.1.2"
-airbyte-cdk = "^4"
+airbyte-cdk = "~4.3"
```

Comment:
why the restriction?

Reply:
I originally added the restriction because of a change made here https://github.com/tcboles/airbyte/commit/fca0460030f1e7d95b8b2471b34844acd63c615e#diff-51c861828e0a614ef6ee21390f3ba17d7138549f0575051fdb24868ee05eabbbL99 which changed the constructor of ConnectorStateManager. I just updated the stripe code for the updated constructor and it seems to work as expected without the old param.